SoftLayer/managers/monitor.py

```diff
@@ -0,0 +1,85 @@
+"""
+    SoftLayer.monitor
+    ~~~~~~~~~~~~~~~~~~
+    Monitoring Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+
+from SoftLayer import utils
+
+
+class MonitoringManager(utils.IdentifierMixin, object):
+    """Manages monitoring for Hardware and Virtual Servers
+
+    :param SoftLayer.API.CLient client: an API client instance
+    :param string server_tpe: should be either
+                  'Hardware_Server' or 'Virtual_Guest'
+    """
+
+    def __init__(self, client, server_type='Hardware_Server'):
```

Comment:
I really don't like the `server_type` context... This makes one instance of the manager not re-usable between hardware and virtual guests... So this, in affect, is a HardwareMonitoringManager or VirtualGuestHardwareManager depending on what the value of server_type is. It's context, and most other managers have no additional context other than the API client.

Regarding how removing this would affect get_status(), this is a case where it probably should be broken into get_status_for_virtual_server() and get_status_for_hardware(), because that's what you're really doing (taking into the server_type context).